src/core/server/services/stories/index.ts

```diff
@@ -368,3 +369,37 @@ export async function merge(
   // Return the story that had the other stories merged into.
   return destinationStory;
 }
+
+/**
+ * updateStoryLastCommentedAt will update the date of lastCommentedAt.
+ *
+ * @param mongo the mongo database
+ * @param tenant the Tenant to operate on
+ * @param story the story that we're setting lastCommentedAt on
+ * @param now the current time
+ */
+export async function updateStoryLastCommentedAt(
+  mongo: Db,
+  tenant: Tenant,
+  story: Story,
+  now: Date
+) {
+  // Create a logger for this operation.
+  const log = logger.child({ tenantID: tenant.id, storyID: story.id }, true);
+
+  const result = await collections.stories(mongo).findOneAndUpdate(
```

Comment:
Database interactions should be contained to the `src/core/server/models` directory. These model functions should always return the result (the story) that was updated.